cluster/addons/hp-client/hugepage-isolator.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/cluster/addons/hp-client/hugepages"
+	"k8s.io/kubernetes/cluster/addons/iso-client/isolator"
+)
+
+const (
+	// kubelet eventDispatcher address
+	eventDispatcherAddress = "localhost:5433"
+	// iso-client own address
+	isolatorLocalAddress = "localhost:5444"
```

Comment:
Can you change your `isolatorLocalAddress` to sth different? It will overlapping with already existing `coreaffinity` isolator.